clients/src/main/java/org/apache/kafka/common/requests/ApiVersionsResponse.java

```diff
@@ -56,6 +56,15 @@ public ApiVersion(short apiKey, short minVersion, short maxVersion) {
             this.minVersion = minVersion;
             this.maxVersion = maxVersion;
         }
+
+        @Override
+        public String toString() {
+            return "ApiVersion {" +
+                    " apiKey: " + apiKey +
+                    " minVersion: " + minVersion +
+                    " maxVersion: " + maxVersion +
+                    "}";
```

Comment:
If you look at `ProducerRecord`, we use a slightly different convention for `toString` (I know we are not consistent everywhere, but I'm trying to improve that).